src/workloads/execution/UnionWith.yml

```diff
@@ -10,12 +10,13 @@ Actors:
     Database: &db test
     Threads: 1
     CollectionCount: 20
-    DocumentCount: 10000
-    BatchSize: 1000
+    DocumentCount: 5000
+    # Choose a batchSize and document size such that all documents can fit in a single 16MB batch.
+    BatchSize: &batchSize 100000
     Document:
-      integer: &integer {^RandomInt: {min: 0, max: 10000}}
+      integer: &integer {^RandomInt: {min: 1, max: 10000}}
```

Comment:
Oh yeah, I should mention that leaving a min: 0 here was causing the $divide in one of the workloads to fail. I'm not sure if this isn't caught by evergreen or if I just got unlucky.

Reply:
Thanks! Turns out the reason is that explain doesn't report any runtime errors in the command being explained, so we were silently ignoring this before.